### PR TITLE
Add /contribute.json

### PR DESCRIPTION
**Pull request by initrd**
Add a /contribute.json in the format suggested at https://www.contributejson.org/

### CODE REVIEW

**Comment by initrd** [comment]
This is for [Bug 1349402](https://bugzilla.mozilla.org/show_bug.cgi?id=1349402)

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "repository": {
+    "url": "https://github.com/mozilla/irc-iplimit",
+    "license": "MPL2"
+  },
+  "keywords": [
+    "Python",
+    "Flask",
+    "JSON"
+  ],
+	"participate": {
```
Comment (floatingatoll) [inline contribute.json]: This whitespace indent pattern doesn't match the rest of the JSON. It won't cause parsing errors, but it does make it harder to read as a human being.

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		"home": "https://iplimit.irc.mozilla.org/",
+    "irc": "ircs://irc.mozilla.org",
```
Comment (floatingatoll) [inline contribute.json]: `ircs://` alone is not sufficient to direct clients to SSL port 6697. Please use `ircs://irc.mozilla.org:6697` here. (Note that while RFC 7194 defines `ircs-u` as the correct protocol for port 6697, not even Firefox recognizes that.)

File: contribute.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "irc-contacts": "ashish"
```
Comment (floatingatoll) [inline contribute.json]: Consider making this either `#moc` or an array containing both your name and `#moc`.

**Comment by initrd** [comment]
Thanks @floatingatoll, I've resolved the concerns in d7dca51.

**Pull request closed without merging**